pkg/gofr/datasource/mongo/mongo_test.go

```diff
@@ -536,16 +536,16 @@ func TestClient_StartSession(t *testing.T) {
 		resp, err := cl.InsertOne(context.Background(), mt.Coll.Name(), doc)
 
 		assert.NotNil(t, resp)
-		assert.Nil(t, err)
+		require.NoError(t, err)
 
 		err = ses.CommitTransaction(context.Background())
 
-		assert.Nil(t, err)
+		require.NoError(t, err)
 
 		ses.EndSession(context.Background())
 
 		// Assert that there was no error
-		assert.Nil(t, err)
+		require.NoError(t, err)
```

Comment:
We are using both `assert` as well as `require` in the test file. We can use any one of them throughout.

Reply:
Do you mean stop using assert?abd only use require package?

It seems a regression. Having tests that fails on first issue is not the best thing

Thi k about checking the http.Status, then the content of the JSON returned

If the check on http.Status fail, you have no idea what the problem is without tjejson content

Reply:
@Umang01-hash thoughts ?

Reply:
ahhh! Got it 
Sorry my bad! Let's use both